tests/Unit/Helpers/Domain/DomainTestHelpers.cpp

```diff
@@ -6,7 +6,9 @@
 #include "Helpers/Domain/DomainTestHelpers.hpp"
 
 #include <algorithm>
+#include <catch.hpp>
```

Comment:
don't need `<catch.hpp>` that gets sucked in through `TestingFramework.hpp` (order matters for that one, unfortunately)